docs/index.html

```diff
@@ -2,69 +2,191 @@
 layout: default
 title: eBay Skin
 ds: 6
+ds_map:
+    alert-dialog:
+        name: alert-dialog
+        version: 2.1
+    avatar:
+        name: avatar
+        version: 1.0
+    badge:
+        name: badge
+        version: 1.2
+    breadcrumbs:
+        name: breadcrumb
+        version: 1.2
+    button:
+        group: button
+        name: button
+        version: 1.6
+    carousel:
+        name: carousel
+        version: 1.1
+    calendar:
+        name: calendar
+        version: 1.0
+    checkbox:
+        name: checkbox
+        version: 2.0
+    chip:
+        version: 1.0.0
+    chips-combobox:
+        name: chip-combobox
+        version: 1.0.0
+    combobox:
+        name: dropdown
+        version: 2.2
+    confirm-dialog:
+        name: alert-dialog
+        version: 2.1
+    cta-button:
+        group: button
+        name: button
+        version: 1.2
+    date-textbox:
+        name: date-picker
+        version: 1.0
+    details:
+        name: expansion
+        version: 1.2
+    drawer-dialog:
+        name: bottom-sheet
+        status: deprecated
+        version: 2.0
+    eek:
+        name: eek
+        version: 1.1
+    filter-button:
+        name: quick-filter
+        version: 2.0
+    filter-menu:
+        version: 2.0
+    filter-menu-button:
+        name: quick-filter
+        version: 2.0
+    flag:
+        version: 1.0
+    floating-label:
+        name: form-field
+        version: 2.2
+    fullscreen-dialog:
+        name: modal
+        version: 2.1
+    icon:
+        name: icon
+        version: 1.3
+    icon-button:
+        group: button
+        name: icon-button
+        version: 1.6
+    infotip:
+        name: infotip
+        version: 2.0
+    inline-notice:
+        name: alert-notice
+        version: 2.2
+    lightbox-dialog:
+        - name: modal
+          version: 2.1
+        - name: bottom-sheet
+          version: 2.1
+    less:
+        version: 1.0
+    link:
+        name: text-link
+        version: 2.0
+    listbox:
+        name: dropdown
+        version: 2.2
+    listbox-button:
+        name: dropdown
+        version: 2.2
+    menu:
+        name: dropdown
+        version: 2.3
+    menu-button:
+        name: dropdown
```

Comment:
The `name` values in `listbox`, `listbox-button`, `menu`, etc. all have `dropdown`. Weren't the names supposed to be unique? Were these supposed to be `group` instead?

Reply:
Alas, there isn't always a 1:1 mapping from Design System to Skin. Currently, the DS considers all of these dropdowns. I suspect we'll expand the DS to have a more nuanced approach to menus vs listboxes, and when that happens, we can update this list accordingly. But in the meantime we want a way to programmatically determine which Skin components implement the "Dropdown" DS pattern.

Reply:
I see. Sounds good.